app/src/main/res/layout-land/profile_edit_activity.xml

```diff
@@ -119,6 +119,7 @@
                 android:layout_height="wrap_content"
                 android:fontFamily="sans-serif"
                 android:text="@string/profile_edit_allow_download_sub"
+                android:textColor="#61000000"
```

Comment:
Please add in colors file and use it here.

Reply:
what should I name the color. Can I name it oppia_transparent_text_color?